File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/account/AccountType.java
package com.example.cf_sdk.logic.changebanksdk.model.account;

import com.google.gson.annotations.SerializedName;

/**
 *
 * Enum for Member Account Type
 */
public enum AccountType {
    UNKNOWN("Unknown"),

    @SerializedName("CHEQUE")
    CHEQUE("Checking"),

    @SerializedName("SAVINGS")
    SAVINGS("Savings");

    String mTypeName;

    AccountType(String typeName) {
        mTypeName = typeName;
    }

    public String getTypeName() {
        return mTypeName;
    }
}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/account/AccountType.kt
package com.example.sdk_no_dagger.changebankapi.model.account

import com.google.gson.annotations.SerializedName

/**
 *
 * Enum for Member Account Type
 */
enum class AccountType(typeName: String) {
    UNKNOWN("Unknown"), @SerializedName("CHEQUE")
    CHEQUE("Checking"), @SerializedName("SAVINGS")
    SAVINGS("Savings");

    var typeName = ""

    init {
        this.typeName = typeName
    }
}